### PR TITLE
Makes mind swapping instant

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Makes mindswap happen instantly, unleashing a loud sound and silencing everyone in a range around both the target. Who can themselves be up to 3 tiles away. This also removes the hardstun, so you can start running instantly.

## Why It's Good For The Game
In a gamemode where many people will be running at each other at once, we need to keep the meta info on things as low as possible. After the wizard loudly yells I AM USING THE MINDSWAP SPELL and two people are stuck jittering on the floor, the wizard is pretty much dead if they aren't careful about it, or if there is at least one other person who can take that 10 second stun to beat the shit out of you.

Let's make it faster :trollface:

## Testing
swapped minds with myself once or twice. effects worked as intended.

## Changelog
:cl:
tweak: Mindswap is now instant at range 3. It now doesn't stun, but it silences folks around in a range and plays an obvious sound.
/:cl:

### CODE REVIEW

**Comment by Migratingcocofruit** [comment]
One thing I'd like to be added is the ability to fake mind swap by targeting yourself, or something of the sort. That way if there are 3 people the one not mind swapped won't know who is who regardless of whether or not they know the spell OOC.

**Comment by Hematoma** [comment]
If wizards had a magical eyes that could see others wizards regardless of body that would be great too.

**Comment by lewcc** [comment]
> One thing I'd like to be added is the ability to fake mind swap by targeting yourself, or something of the sort. That way if there are 3 people the one not mind swapped won't know who is who regardless of whether or not they know the spell OOC.

I love that idea, done!

**Review by Sirryan2002 — APPROVED** [review]

File: code/datums/spells/mind_transfer.dm [hunk 1]
```diff
@@ -38,44 +41,47 @@ Also, you never added distance checking after target is selected. I've went ahea
...
+				add_verb(caster, V)
 	//MIND TRANSFER END
 
-	//Here we paralyze both mobs and knock them out for a time.
-	caster.Paralyse(paralysis_amount_caster)
-	victim.Paralyse(paralysis_amount_victim)
+	for(var/mob/new_or_formal_wizard in list(victim, caster))
+		for(var/mob/living/L in range(3, new_or_formal_wizard))
+			L.SetSilence(15 SECONDS)
+
```
Comment (DGamerL) [inline code/datums/spells/mind_transfer.dm:86]:
```suggestion
```

**Review by DGamerL — APPROVED** [review]

**Pull request merged**